src/control/dmg/main.go

```diff
@@ -48,10 +53,25 @@ type cliOptions struct {
 }
 
 var (
-	opts  = new(cliOptions)
-	conns = client.NewConnect()
+	opts                = new(cliOptions)
+	conns               = client.NewConnect()
+	rootCertName string = "daosCA.crt"
+	certName     string = "shell.crt"
+	keyName      string = "shell.key"
 )
 
+func loadCredentials() (credentials.TransportCredentials, error) {
+	if opts.Insecure {
+		fmt.Println("Using Insecure connection")
+		return nil, nil
+	}
+	caRootPath := opts.CertDir + "/" + rootCertName
```

Comment:
is there an equivalent of python's os.path.join we can do rather than manually inserting separator?

Reply:
I can move all of this over to using FilePath or Path I believe and there are ways to append to paths with those objects.